src/containers/search_result.jsx

```diff
@@ -298,7 +281,33 @@ class SearchResult extends Component {
   renderSearchResults() {
     const { search: { visibility } } = this.props;
 
-    const renderHeaderColumn = (options) => {
+    // const renderHeaderColumn = (options) => {
+    //   const opts = {
+    //     ...{
+    //       key: '',
+    //       label: '',
+    //       forceShow: false,
+    //     },
+    //     ...options,
+    //   };
+
+    //   const { key, label, forceShow } = opts;
+
+    //   return (
+    //     (forceShow || visibility[key]) && <TableHeaderColumn
+    //       style={{
+    //         ...styles.column,
+    //         flex: flex[key],
+    //       }}
+    //       title={label}
+    //     >
+    //       {label}
+    //     </TableHeaderColumn>
+    //   );
+    // };
```

Comment:
Do not comment the previous implementation. If you want to make some changes, just refactor the existing code and merge your new part into it.

Reply:
finished